app/src/main/java/com/sdex/activityrunner/manifest/ManifestViewerActivity.kt

```diff
@@ -96,11 +98,37 @@ class ManifestViewerActivity : BaseActivity() {
         }
 
         viewModel.loadManifest(appPackageName)
+        setupFindInPage()
+        onBackPressedDispatcher.addCallback(object : OnBackPressedCallback(true) {
+            override fun handleOnBackPressed() {
+                isEnabled = false
```

Comment:
If I open the search a second time, the back button closes the activity instead of searching. Probably, the dispatcher should always be enabled.